### PR TITLE
updating csv file columns after talking with ux

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
After discussion with UX, I've put in a filter to only render specific columns in a specific order to make this more user-friendly. Please see previous PR #2422 for origin.

### CODE REVIEW

**Review by Smithh-Co — APPROVED** [review]
Ty for the quick action Andrew! LGTM

File: tdrs-backend/tdpservice/parsers/models.py [hunk 1]
```diff
@@ -25,8 +25,8 @@ class Meta:
     item_number = models.IntegerField(null=False)
     field_name = models.TextField(null=False, max_length=128)
     category = models.IntegerField(null=False, default=1)
-    case_number = models.TextField(null=True, max_length=128)
-    rpt_month_year = models.IntegerField(null=True, blank=False)
+    rpt_month_year = models.TextField(null=True, max_length=128)
+    case_number = models.IntegerField(null=True, blank=False)
```
Comment (andrew-jameson) [inline tdrs-backend/tdpservice/parsers/models.py:30]: @raftmsohani It seems to me we overlooked this previously. This should have been flipped, yes?

File: tdrs-backend/tdpservice/parsers/views.py [hunk 2]
```diff
@@ -38,12 +38,23 @@ def _get_xls_serialized_file(self, data):
...
         worksheet = workbook.add_worksheet()
 
+        report_columns = ['case_number',
+                          'rpt_month_year',
+                          'error_type',
+                          'error_message',
+                          'item_number',
+                          'field_name',
+                          'row_number',
+                          'column_number']
```
Comment (andrew-jameson) [inline tdrs-backend/tdpservice/parsers/views.py:48]: @reitermb @lilybl1 Are these columns more inline with your vision?

**Review by raftmsohani — APPROVED** [review]
LGTM!

**Review by reitermb — APPROVED** [review]
Changes look great to me—thanks for the super quick turnaround too!

File: tdrs-backend/tdpservice/parsers/migrations/0002_auto_20230403_1934.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    operations = [
+        migrations.AlterField(
+            model_name='parsererror',
+            name='case_number',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='parsererror',
+            name='rpt_month_year',
+            field=models.TextField(max_length=128, null=True),
```
Comment (raftmsohani) [inline tdrs-backend/tdpservice/parsers/migrations/0002_auto_20230403_1934.py]: rpt_month_year should be 'int', see https://github.com/raft-tech/TANF-app/blob/223e5f115fa304cfd534790adae18a65c10793ec/tdrs-backend/tdpservice/search_indexes/models.py#L19

**Review by raftmsohani — CHANGES_REQUESTED** [review]
I made a new comment for 'rpt_month_year'

**Review by raftmsohani — APPROVED** [review]
Looks good to me :)

File: tdrs-backend/tdpservice/parsers/models.py [hunk 4]
```diff
@@ -25,8 +25,8 @@ class Meta:
     item_number = models.IntegerField(null=False)
     field_name = models.TextField(null=False, max_length=128)
     category = models.IntegerField(null=False, default=1)
-    case_number = models.TextField(null=True, max_length=128)
-    rpt_month_year = models.IntegerField(null=True, blank=False)
+    rpt_month_year = models.IntegerField(null=True,  blank=False)
+    case_number = models.IntegerField(null=True, blank=False)
```
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/models.py]: @andrew-jameson @reitermb @lilybl1 case number can have alphanumeric characters.

**Review by ADPennington — APPROVED** [review]
👍🏾

**Pull request merged**